src/zenml/cli/__init__.py

```diff
@@ -372,6 +372,47 @@
 zenml secret delete SECRET_NAME
 ```
 
+Interacting with Deployed Models
```

Comment:
It's not a bad idea, I don't know why it didn't occur to us before... @AlexejPenner , any thoughts ?

Reply:
Good question. I think the advantage would be a CLI with less root groups - however conceptually I feel it makes more sense for the user to interact directly with the user facing entities (secrets, models) rather than the entity management (secret-manager, model-deployer). Interaction with the entity management would be reserved for their configuration.  Happy to talk this through though